fusionfrontend/src/app/factory/models/status.model.ts

```diff
@@ -14,12 +14,18 @@
  */
 
 import { OispDeviceStatus } from '../../core/models/kairos.model';
+import { ID } from '@datorama/akita';
 
 export class Status {
   gotData: boolean;
   statusValue: string;
 }
 
+export class StatusWithAssetId {
+  factoryAssetId: ID;
```

Comment:
why not extending FactoryAsset? That way you could pass a single data array to the corresponding components